aten/src/ATen/native/quantized/cpu/quant_utils.h

```diff
@@ -0,0 +1,117 @@
+#pragma once
+
+#include <ATen/ATen.h>
+#include <algorithm>
+#include <cmath>
+
+namespace quant_utils {
+using namespace std;
+// A structure to hold quantization parameters 'scale' and 'zero_point'.
+// The meaning of these values is as the constants in the quantization equation
+//
+//   real_value = scale * (quantized_value - zero_point)
+//
+// In other words, 'zero_point' is the quantized value that corresponds
+// to the real value 0, and 'scale' is the difference of real values
+// corresponding to consecutive quantized values.
+struct TensorQuantizationParams {
+  float scale;
+  std::int32_t zero_point;
+  int precision;
+};
+
+inline TensorQuantizationParams ChooseQuantizationParams(
```

Comment:
Want to point out that ChooseQuantizationParams in FBGEMM (https://github.com/pytorch/FBGEMM/blob/master/src/QuantUtils.cc#L19) is originally from GEMMLowp:
https://github.com/google/gemmlowp/blob/master/doc/quantization_example.cc#L71